content/blog/coredns-in-kubernetes-via-kubeadm.md

```diff
@@ -53,34 +75,23 @@ Use the following command to install CoreDNS as default while installing a fresh
 
 Your Kubernetes master has initialized successfully!
 ~~~
+CoreDNS install is confirmed if we see the following output while deploying Kubernetes.
+~~~
+[addons] Applied essential addon: CoreDNS
+~~~
 
 ## Updating your existing cluster to use CoreDNS
 In case you have an existing cluster, it is also possible to upgrade your DNS to CoreDNS, replacing kube-dns using the `kubeadm upgrade` command. 
 
-Using `kubeadm upgrade plan` and by setting `feature-gates` flag as `true`, it is possible to check the CoreDNS version that will be installed before proceeding to apply the changes.
+Using `kubeadm upgrade plan` and by setting `feature-gates` flag as `CoreDNS=true`, it is possible to check the CoreDNS version that will be installed before proceeding to apply the changes.
 
 Checking the CoreDNS version to upgrade:
-
+~~~ text
+# kubeadm upgrade plan --feature-gates CoreDNS=true
+~~~
 ~~~ text
 # kubeadm upgrade plan --allow-experimental-upgrades --feature-gates CoreDNS=true
```

Comment:
I guess you need to be consistent with info 2 lines above or explain why you add the "--allow-experimental-upgrades"